pac4j-jwt/src/main/java/org/pac4j/jwt/profile/JwtGenerator.java

```diff
@@ -39,9 +39,15 @@
     protected final Logger logger = LoggerFactory.getLogger(getClass());
 
     private final String secret;
+    private final boolean encrypted;
```

Comment:
Encryption should happen by default for security reasons and backward compatibility.